EnglishFilter/sections/antiadblock.txt

```diff
@@ -495,9 +495,10 @@ elpais.com,definitions.net,ensonhaber.com,lapatilla.com,sozcu.com.tr#%#//scriptl
 ||cdn.staticaly.com/gh/Arlina-Design/*/arlinablock.js
 ||rawcdn.githack.com/*/arlinablock.js
 ||cdn.jsdelivr.net/gh/dewaplokis/block@master/dewablock.js
+||cdn.jsdelivr.net/gh/*/js/arlinablock.js
 iptvbin.com$$script[tag-content="a.id=""arlinablock"""][max-length="1200"]
-||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,redirect=googlesyndication-adsbygoogle,important,domain=drivedrama.com|puressh.net|iptvbin.com
-@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=drivedrama.com|puressh.net|iptvbin.com
+||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,redirect=googlesyndication-adsbygoogle,important,domain=gocmod.com|drivedrama.com|puressh.net|iptvbin.com
+@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=gocmod.com|drivedrama.com|puressh.net|iptvbin.com
```

Comment:
Why did you add the domain to those rules?

Reply:
@zloyden It's needed for the anti-adblock to be wiped out.

Reply:
Okay it's needed. Why did I even add it there? I don't have any idea on why did I add it there.